### PR TITLE
Adapter finished

### PR DESCRIPTION
**Pull request by asmei1**

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapterToJobs2dDriver.java [hunk 1]
```diff
@@ -6,13 +6,14 @@
 import edu.kis.powp.jobs2d.Job2dDriver;
 
 /**
- * driver adapter to drawer with several bugs.
+ * Driver adapter to drawer.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawerAdapterToJobs2dDriver implements Job2dDriver {
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapterToJobs2dDriver.java]: Not the other way around?  Jobs2dDriverToDrawerAdapter? 
BTW: The name does not need to mention the adapter pattern (it is not the design issue but only solution to the problem).

File: src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java [hunk 2]
```diff
@@ -83,7 +84,7 @@ public static void main(String[] args) {
 			public void run() {
 				Application app = new Application("2d jobs Visio");
 				DrawerFeature.setupDrawerPlugin(app);
-				setupDefaultDrawerVisibilityManagement(app);
+				//setupDefaultDrawerVisibilityManagement(app);
```
Comment (kowallus) [inline src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java]: It is safe to delete in revision repo

File: src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java [hunk 3]
```diff
@@ -7,15 +7,22 @@
...
 	public SelectTestFigureOptionListener(DriverManager driverManager) {
 		this.driverManager = driverManager;
 	}
-
+	
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		switch (e.getActionCommand()) {
+			case "Figure Joe 1":
```
Comment (kowallus) [inline src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java]: Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.

**Comment by kamsykula** [comment]
Should I fix this and make a new pull request or leave it like now?

File: src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapterToJobs2dDriver.java (hunk 1 above)
Comment (asmei1) [reply src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapterToJobs2dDriver.java]: Ok, now should be fixed.

File: src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java (hunk 2 above)
Comment (asmei1) [reply src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java]: Yes, I miss that.

File: src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java (hunk 3 above)
Comment (asmei1) [reply src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java]: Ok, I fixed this problem with enum

File: src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawController.java [hunk 4]
```diff
@@ -6,13 +6,14 @@
 import edu.kis.powp.jobs2d.Job2dDriver;
 
 /**
- * driver adapter to drawer with several bugs.
+ * Driver adapter to drawer.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawController implements Job2dDriver {
```
Comment (rsusik) [inline src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawController.java:11]: The name does not indicate the purpose of the adapter or what was actually adapted.

**Review by rsusik — COMMENTED** [review]
Specific remarks are inlined in the code.

**Pull request closed without merging**